modelci/types/bo/model_objects.py

```diff
@@ -152,30 +152,45 @@ def __str__(self):
 
 
 class InfoTuple(object):
-    """A triplet tuple containing min, max and average values of a data over a period of time.
+    """A triplet tuple containing overall, average, 50th percentile, 95th percentile, and 99th percentile values of a
+    data over a period of time.
+
+    Args:
+        info (Tuple[float, float, float, float]): a triplet containing average, 50th percentile, 95th percentile,
+        and 99th percentile values,  of a data.
     """
 
-    def __init__(self, info: Tuple[float, float, float]):
+    def __init__(self, info: Tuple[float, float, float, float]):
```

Comment:
passing tuple here has a poor readability, since we don't know `(1, 1, 1, 1)` what are them in the parameter list if we don't check your source code. A way out is something like

```suggestion
    def __init__(self, avg: float, p50: float, p95:float, p99:float):
```
and init the tuple in its constructor.